modelutils.py

```diff
@@ -17,13 +19,23 @@ def skip(*args, **kwargs):
 
     saved_inits = torch.nn.init.kaiming_uniform_, torch.nn.init.uniform_, torch.nn.init.normal_  # preserving
```

Comment:
if refactoring this functions, I suggest to use context manager for cleaner code:

```
from contextlib import contextmanager

@contextmanager
def suspend_nn_inits():
    skip = lambda *args, **kwargs: None  
    saved_inits = torch.nn.init.kaiming_uniform_, torch.nn.init.uniform_, torch.nn.init.normal_  #saving
    torch.nn.init.kaiming_uniform_ = torch.nn.init.uniform_ = torch.nn.init.normal_ = skip  #replacing     
    try:
        yield
    finally:
        torch.nn.init.kaiming_uniform_, torch.nn.init.uniform_, torch.nn.init.normal_ = saved_inits  # restoring
```

Reply:
I will update read me file.

Reply:
with suspend_nn_init():
    model = ...
    do_normal_initalization_as_usual()

Reply:
See updated code above with refactored function

Reply:
I decided to use the code that you provided. Thank you!